src/custom/pages/Faq/index.tsx

```diff
@@ -745,6 +745,44 @@ export default function Faq() {
         */}
         </Content>
       </Page>
+      <Page>
+        <Content>
+          <h2 id="stats-from-blockchain">Stats from Blockchain</h2>
```

Comment:
Now that we have the rest of the FAQ, makes more sense to point this PR to it https://github.com/gnosis/cowswap/pull/1691 and remove this new section

Reply:
Do you mean that Stats from Blockchain isn't a new section anymore? Should I move the content inside the Affiliate program section?

Reply:
Yes!